docs/Configuration.md

```diff
@@ -201,7 +201,7 @@ The full url appears then as `redis://user:passwd@1.2.3.4:6379/?db=1`.
 
 #### gha
 
-* `SCCACHE_GHA_CACHE_URL` / `ACTIONS_CACHE_URL` GitHub Actions cache API URL
+* `SCCACHE_GHA_CACHE_URL` / `ACTIONS_RESULTS_URL` GitHub Actions cache API URL
```

Comment:
What about backward compatibility of the variable names? Your changes break it. Is it what you really want?